pylatex/__init__.py

```diff
@@ -5,16 +5,21 @@
     :license: MIT, see License for more details.
 """
 
+from .basic import HugeText, NewPage, LineBreak, NewLine, HFill, LargeText, \
+    MediumText, SmallText, FootnoteText, TextColor
 from .document import Document
+from .frames import MdFramed, FBox
 from .math import Math, VectorName, Matrix
 from .package import Package
 from .section import Section, Subsection, Subsubsection
 from .table import Table, MultiColumn, MultiRow, Tabular, Tabu, LongTable, \
-    LongTabu
+    LongTabu, Tabularx, Column
 from .tikz import TikZ, Axis, Plot
-from .figure import Figure, SubFigure
+from .figure import Figure, SubFigure, StandAloneGraphic
 from .lists import Enumerate, Itemize, Description
 from .quantities import Quantity
 from .base_classes import Command, UnsafeCommand
-from .utils import NoEscape, escape_latex
+from .utils import NoEscape, escape_latex, _latex_item_to_string
```

Comment:
Why did you export this? It should not be used outside of the library. Did you need it for something, because that might function be useful to add to the utils then.